routes/index.ts

```diff
@@ -39,4 +39,5 @@ app.use("/v1/notifications", require("./notify"));
 app.use("/goals", require("./goals"));
 app.use("/invites", require("./invites"));
 app.use("/requests", require("./requests"));
+app.use("/subscription", require("./subscription"));
```

Comment:
- should this be `subscription` or `subscriptions`?

Reply:
subscription sir